### ISSUE TITLE
Adrenal agenesis

### ISSUE THREAD

**Issue by poratshay** [obophenotype/human-phenotype-ontology#11565]
**Preferred term label:**
Adrenal agenesis

**Synonyms**
Bilateral congenital adrenal agenesis
Congenital absence of the adrenal glands

**Definition (free text, please give PubMed ID)**
Congenital adrenal agenesis is an extremely rare condition wherein the adrenal glands fail to develop, resulting in complete absence of adrenal tissue and the hormones produced in the adrenal cortex and medulla (PMID: 24875163)

**Parent term (use hpo.jax.org/app)**
Adrenal gland

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0001-9947-035X

**Comment by pnrobinson** [comment]
@poratshay  This was:
[Adrenal gland agenesis](https://hpo.jax.org/browse/term/HP:0011743)
I will add the above synonyms! Thanks
